projects/schematics/src/shared/constants.ts

```diff
@@ -639,3 +648,4 @@ export const CSS_V2_DOCS_URL =
 export const CSS_V3_DOCS_URL =
   'https://sap.github.io/spartacus-docs/css-changes-in-version-3/';
 /***** Storefinder end *****/
+export const CLI_QUALTRICS_FEATURE = 'Qualtrics';
```

Comment:
Please remove this, as you don't have any CLI options  for Qualtrics' schematics